lte/protos/subscriberdb.proto

```diff
@@ -242,7 +242,7 @@ message SyncSubscribersResponse {
   repeated SubscriberDigestWithID per_sub_digests = 3;
   // to_renew is the set of all subscriber data modified/added on the server side,
   // indexed by their IDs (IMSI).
-  map<string, SubscriberData> to_renew = 4;
+  repeated SubscriberData to_renew = 4;
```

Comment:
1. This is mostly for consistently. `SyncSubscribers` and `ListSubscribers` both read their results from the subscriber proto cache, which returns an ordered list, and I thought it was unnecessary to convert the list in `SyncSubscribers` to a map, especially since on the AGW side a list works the same way as a map
2. Yep! I was waiting to make that change either here or in the agw `SyncSubscribers` PR, depending on which one gets merged first @hcgatewood

Reply:
Cool. Make the AGW change in this PR as well then, so nothing breaks in-between commits

Reply:
Yep, will have to wait until the code for `SyncSubscribers` in AGW come in from the other PR though @hcgatewood